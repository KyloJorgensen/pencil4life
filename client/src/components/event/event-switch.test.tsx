'use strict';

import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import EventSwtich from './event-switch';

let wrapper: ShallowWrapper;

describe('Render EventSwtich', () => {
	it('matches the snapshot', () => {
		wrapper = shallow(
			<EventSwtich/>
		);
		expect(wrapper).toMatchSnapshot();
	});
});