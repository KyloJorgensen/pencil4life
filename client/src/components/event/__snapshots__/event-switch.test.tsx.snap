// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render EventSwtich matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventProvider />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <EventSwitch />,
      "initialState": Object {
        "eventItemList": Object {},
        "eventItems": Array [],
        "limit": 0,
        "page": 0,
        "pageTotal": 0,
        "total": 0,
        "totalPages": 0,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <EventSwitch />,
        "initialState": Object {
          "eventItemList": Object {},
          "eventItems": Array [],
          "limit": 0,
          "page": 0,
          "pageTotal": 0,
          "total": 0,
          "totalPages": 0,
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
